src/w3iProxy/index.ts

```diff
@@ -163,6 +183,13 @@ class Web3InboxProxy {
 
     this.initializing = true
 
+    if (this.core) {
```

Comment:
Can we move this to here (https://github.com/WalletConnect/web3inbox/pull/220/files#diff-82bbad320eb11fb8fe700ee91086488459dc806d83d3298899fbecf6ab15da64R96) where the core is created conditionally, instead of creating a separate if check?

Reply:
It would end being a long, kind of unreadable promise chain so I went with this. If this is a strong preference I can do that